src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -71,6 +73,11 @@ interface DeleteWorkspaceAction extends Action {
   workspaceId: string;
 }
 
+interface TerminateWorkspaceAction extends Action {
+  type: 'DEV_TERMINATE_WORKSPACE';
```

Comment:
Is this DEV means dev event or DevWorkspaces. If the second I would rename it like the following
```suggestion
  type: 'TERMINATE_DEVWORKSPACE';
```

Reply:
I have fixed it

Reply:
@olexii4 I'd ask you to rename all remaining actions to keep their consistency.

Reply:
I have fixed it